includes/class-wc-stripe-api.php

```diff
@@ -16,6 +16,28 @@ class WC_Stripe_API {
 	const ENDPOINT           = 'https://api.stripe.com/v1/';
 	const STRIPE_API_VERSION = '2024-06-20';
 
+
+	/**
+	 * Option key for cases where Stripe rate limits our live API calls.
+	 *
+	 * @var string
+	 */
+	public const LIVE_MODE_STRIPE_API_RATE_LIMIT_OPTION_KEY = 'wc_stripe_live_api_rate_limit';
+
+	/**
+	 * Option key for cases where Stripe rate limits our test API calls.
+	 *
+	 * @var string
+	 */
+	public const TEST_MODE_STRIPE_API_RATE_LIMIT_OPTION_KEY = 'wc_stripe_test_api_rate_limit';
+
+	/**
+	 * Duration we will use to disable Stripe API calls if we have been rate limited.
+	 *
+	 * @var int
+	 */
+	public const STRIPE_API_RATE_LIMIT_DURATION = 30;
```

Comment:
Is this seconds? Or minutes?

At first I thought it would be minutes, but from the code below, it turned out to be seconds.

Reply:
That is fair. If you had to think twice, the name needs to be better! I've added `_IN_SECONDS` to the constant name in 84e5af28 so it's super-explicit.